scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -309,16 +314,20 @@ abstract class LocalScanner(
 
         if (missingArchives.isEmpty()) return
 
+        val tempDirectory = createTempDirectory().toFile()
+
```

Comment:
Would it make sense to call "deleteOnExit()" to ensure deletion if code throws exception inbetween directory creation and deletion commands?

Reply:
I started to avoid calling `deleteOnExit()` as I learned it registers static handlers in the JVM that might leak memory. Also calling both `safeDeleteRecursively()` and `deleteOnExit()` seems a bit involved to me, AFAIK we do that nowhere else in the code base yet. Finally, you may regard this as an implicit "keep temporary files on errors" feature 😉